keyboards/wolf/xox70/solder/info.json

```
@@ -0,0 +1,135 @@
+                { "matrix": [1, 7], "x": 7.5, "y": 1 },
+                { "matrix": [1, 8], "x": 8.5, "y": 1 },
+                { "matrix": [1, 9], "x": 9.5, "y": 1 },
+                { "matrix": [1, 10], "x": 10.5, "y": 1 },
+                { "matrix": [1, 11], "x": 11.5, "y": 1 },
+                { "matrix": [1, 12], "x": 12.5, "y": 1 },
+                { "matrix": [1, 13], "w": 1.5, "x": 13.5, "y": 1 },
+                { "matrix": [1, 14], "x": 15.39, "y": 1 },
+                { "matrix": [1, 15], "x": 16.39, "y": 1 },
+                { "matrix": [1, 16], "x": 17.39, "y": 1 },
+                { "matrix": [2, 0], "w": 1.75, "x": 0, "y": 2 },
+                { "matrix": [2, 1], "x": 1.75, "y": 2 },
+                { "matrix": [2, 2], "x": 2.75, "y": 2 },
+                { "matrix": [2, 3], "x": 3.75, "y": 2 },
+                { "matrix": [2, 4], "x": 4.75, "y": 2 },
+                { "matrix": [2, 5], "x": 5.75, "y": 2 },
+                { "matrix": [2, 6], "x": 6.75, "y": 2 },
+                { "matrix": [2, 7], "x": 7.75, "y": 2 },
+                { "matrix": [2, 8], "x": 8.75, "y": 2 },
+                { "matrix": [2, 9], "x": 9.75, "y": 2 },
+                { "matrix": [2, 10], "x": 10.75, "y": 2 },
+                { "matrix": [2, 11], "x": 11.75, "y": 2 },
+                { "matrix": [2, 12], "x": 19.5, "y": 2 },
+                { "matrix": [2, 13], "w": 2.25, "x": 12.75, "y": 2 },
+                { "matrix": [3, 0], "w": 1.25, "x": 0, "y": 3 },
+                { "matrix": [3, 1], "x": 1.25, "y": 3 },
+                { "matrix": [3, 2], "x": 2.25, "y": 3 },
+                { "matrix": [3, 3], "x": 3.25, "y": 3 },
+                { "matrix": [3, 4], "x": 4.25, "y": 3 },
+                { "matrix": [3, 5], "x": 5.25, "y": 3 },
+                { "matrix": [3, 6], "x": 6.25, "y": 3 },
+                { "matrix": [3, 7], "x": 7.25, "y": 3 },
+                { "matrix": [3, 8], "x": 8.25, "y": 3 },
+                { "matrix": [3, 9], "x": 9.25, "y": 3 },
+                { "matrix": [3, 10], "x": 10.25, "y": 3 },
+                { "matrix": [3, 11], "x": 11.25, "y": 3 },
+                { "matrix": [3, 12], "w": 1.75, "x": 12.25, "y": 3 },
+                { "matrix": [3, 13], "x": 14, "y": 3 },
+                { "matrix": [3, 15], "x": 16.39, "y": 3 },
+                { "matrix": [4, 0], "w": 1.5, "x": 0, "y": 4 },
+                { "matrix": [4, 1], "x": 1.5, "y": 4 },
+                { "matrix": [4, 2], "w": 1.5, "x": 2.5, "y": 4 },
+                { "matrix": [4, 3], "w": 3, "x": 4, "y": 4 },
+                { "matrix": [4, 4], "w": 7, "x": 4, "y": 4 },
+                { "matrix": [4, 9], "w": 3, "x": 8, "y": 4 },
+                { "matrix": [4, 10], "w": 1.25, "x": 10, "y": 5.25 },
+                { "matrix": [4, 11], "w": 1.5, "x": 11, "y": 4 },
+                { "matrix": [4, 12], "x": 12.5, "y": 4 },
+                { "matrix": [4, 13], "w": 1.5, "x": 13.5, "y": 4 },
```

Cleans up the Configurator rendering on the bottom row.

```suggestion
                { "matrix": [4, 0], "w": 1.25, "x": 0, "y": 4 },
                { "matrix": [4, 1], "w": 1.25, "x": 1.25, "y": 4 },
                { "matrix": [4, 2], "w": 1.25, "x": 2.5, "y": 4 },
                { "matrix": [4, 3], "w": 2, "x": 3.75, "y": 4 },
                { "matrix": [4, 4], "w": 2.25, "x": 5.75, "y": 4 },
                { "matrix": [4, 9], "w": 2, "x": 8, "y": 4 },
                { "matrix": [4, 10], "w": 1.25, "x": 10, "y": 5.25 },
                { "matrix": [4, 11], "w": 1.25, "x": 11.25, "y": 4 },
                { "matrix": [4, 12], "w": 1.25, "x": 12.5, "y": 4 },
                { "matrix": [4, 13], "w": 1.25, "x": 13.75, "y": 4 },
```